src/pkg/services/m365/sites.go

```diff
@@ -65,7 +65,11 @@ func SiteByID(
 		Expand: []string{"drive"},
 	}
 
-	s, err := ac.Sites().GetByID(ctx, id, cc)
+	return getSiteByID(ctx, ac.Sites(), id, cc)
+}
+
+func getSiteByID(ctx context.Context, ga api.GetByIDer[models.Siteable], id string, cc api.CallConfig) (*Site, error) {
```

Comment:
corso code standards prefer multiline func props for readability.
```suggestion
func getSiteByID(
    ctx context.Context,
    ga api.GetByIDer[models.Siteable],
    id string,
    cc api.CallConfig,
) (*Site, error) {
```